src/Ui/src/hooks/useGeohash.ts

```diff
@@ -0,0 +1,41 @@
+import {geohashQueryBounds} from "geofire-common";
+import { ListUserParams, Position, User } from "@domain/entities";
+import { useEffect, useState } from "react";
+
+const RADIUS_IN_METERS = 10000
+
+type useGeohashType = {
+  position?: Position,
+  callback: (listUserParams: ListUserParams) => Promise<User[]>
+}
+
+export const useGeohash = ({position, callback}: useGeohashType) => {
```

Comment:
Hey, is not really clear what's the responsibility of this function. Is it the one that is getting the user in a radius of 10km?

Reply:
Renomei o hook para useFilterUsers